doc/tools/docgram/common.edit_mlg

```diff
@@ -1239,6 +1266,9 @@ command: [
 | REPLACE "Preterm" "of" ident
 | WITH "Preterm" OPT ( "of" ident )
 | DELETE "Preterm"
+| REPLACE "Proof" "using" section_var_expr "with" Pltac.tactic
+| WITH "Proof" "using" section_subset_expr OPT [ "with" ltac_expr5 ]
+| DELETE "Proof" "using" section_var_expr
```

Comment:
Instead of making this change in `common.edit_mlg`, I suggest following more closely the implementation for once. I.e. only document `Proof using` (without mentioning the `with` clause) in the proof mode chapter, and put the two forms with the `with` clause in the `auto.rst` file.

Reply:
Note that doc_grammar complains if the same command is shown in more than one :cmd:.  I'll use :cmdv: but then doc_grammar won't check or update it.  And I'll add a reference from Print using to Print with.

A bit of a weird case.  We should start using the pattern `LIST0 [ opt1 | opt2 | ... | optn ]` in the grammar where needed to accept multiple options in any order.